### PR TITLE
Atom : Replaced Uparrow string with its ASCII code

### PR DESCRIPTION
**Pull request by navneetzz**
What I did ? 
Replaced Up arrow and Down arrow strings with their ASCII code.
Added brackets around or terms. 

https://duck.co/ia/view/atom_cheat_sheet
Maintainer: @typo

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Atom Cheat Sheet](https://duck.co/ia/view/atom_cheat_sheet)

**Description**: Atom text editor shortcuts

**Example Query**: [atom cheat sheet](https://beta.duckduckgo.com/?q=atom cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: http://www.shortcutworld.com/en/linux/Atom-(text-editor)_1.0.html

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/atom_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/atom_cheat_sheet)*

File: share/goodie/cheat_sheets/json/atom.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
 			"key": "[Ctrl] [Enter]"
 		}, {
 			"val": "Indent / outdent selected lines",
-			"key": "[Ctrl] [\\[] / [\\]]"
+			"key": "[Ctrl] ([\\[] / [\\]])"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/atom.json]: Could you fix the indentation too? Convert the tabs to 4 spaces.

File: share/goodie/cheat_sheets/json/atom.json [hunk 2]
```diff
@@ -19,124 +19,124 @@
 	],
 	"sections": {
 		"General": [{
-			"val": "Preferences",
-			"key": "[Ctrl] [,]"
+			    "val": "Preferences",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/atom.json]: Can we get these tabs converted to spaces?

**Comment by moollaza** [comment]
@navneetzz looks like this has some conflicts, can you please fix them?

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Comment by navneetzz** [comment]
@jdorweiler I would like to continue and resolve the issue if you open the PR again

**Comment by mintsoft** [comment]
@navneetzz This PR has been reopened; I think it just needs the conflicts resolving then it's ready to be merged

**Comment by GuiltyDolphin** [comment]
@typo You happy with this?

**Comment by navneetzz** [comment]
@GuiltyDolphin I'm not getting it. What is the conflict ?

**Comment by mintsoft** [comment]
@NaveenKarippai if you rebase; you'll see the conflict appear

**Comment by NaveenKarippai** [comment]
@mintsoft you tagged the wrong person :smile:

**Comment by mintsoft** [comment]
@NaveenKarippai I blame github! I typed nav and hit tab; why it autocompleted you I don't know!

**Comment by GuiltyDolphin** [comment]
@mintsoft I find just doing the `@` then `<TAB>` usually works for the author of the PR - not sure why it likes to go off on a tangent if you try and be more specific.

**Comment by NaveenKarippai** [comment]
@mintsoft no worries!

**Comment by moollaza** [comment]
@navneetzz have you tried rebasing your branch yet?

**Comment by navneetzz** [comment]
@moollaza I tried rebasing it a while ago but no difference in status

**Comment by navneetzz** [comment]
@moollaza so that means this PR is of no use anymore?

**Comment by moollaza** [comment]
@navneetzz you don't necessarily need to rebase -- you can just do a `git pull` and resolve the conflicts.

> @moollaza so that means this PR is of no use anymore?

@navneetzz not at all! It just means that 1 or more lines in the Atom Cheat Sheet were change between the time you opened this PR and now. So the problem is that your PR is changing content which has already changed, which Git can't allow.

@navneetzz I've pushed your update code with the conflict resolve to a new PR so you can see the final result: #3152

All I did was checkout your branch, `git pull origin master` (in your case something like `git pull upstream master`). `git status` tells me there's conflicts in `atom.json` -- looking at the code you can see that ther indentation has changed along with some minor changes. I removed the old code, to keep what your PR has and committed that.

**Pull request merged**